external/ILLink/ILLink.depproj

```diff
@@ -1,13 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <NuGetTargetMoniker>.NETCoreApp,Version=v1.1</NuGetTargetMoniker>
-    <NuGetTargetMonikerShort>netcoreapp1.1</NuGetTargetMonikerShort>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
     <OutputPath>$(ToolsDir)ILLink</OutputPath>
     <EnableBinPlacing>false</EnableBinPlacing>
-    <NugetRuntimeIdentifier>$(ToolRuntimeRID)</NugetRuntimeIdentifier>
```

Comment:
What changed that made us not need the RID specific assets?

Reply:
This change from a year ago https://github.com/dotnet/corefx/commit/4c540ae9a9525196390a32c96a1f0bc76e33969a.

I went through most of these projects and tried to clean them up where I noticed stuff that wasn't necessary.